src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,126 @@
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Incorrect get query", e);
+        }
+    }
+
+    @Override
+    public List<Product> getAll() {
+        String query = "SELECT * FROM products";
+        List<Product> productList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            ResultSet resultSet = preparedStatement.executeQuery(query);
+            while (resultSet.next()) {
+                long productId = resultSet.getLong("product_id");
+                Product product = productFromSet(resultSet);
+                product.setId(productId);
+                productList.add(product);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Incorrect getAll query", e);
+        }
+        return productList;
+    }
+
+    @Override
+    public Product update(Product product) {
+        String query = "UPDATE products SET product_name = ?, product_price = ? "
+                + "WHERE product_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, product.getName());
+            preparedStatement.setDouble(2, product.getPrice());
+            preparedStatement.setLong(3, product.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Incorrect update query", e);
+        }
+        return product;
+    }
+
+    @Override
+    public boolean deleteById(Long id) {
+        String query = "DELETE FROM products WHERE product_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, id);
+            return preparedStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Incorrect deleteById query", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Product product) {
+        String query = "DELETE FROM products WHERE product_name = ? AND product_price = ?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, product.getName());
+            preparedStatement.setDouble(2, product.getPrice());
+            return preparedStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Incorrect delete query", e);
+        }
+    }
+
+    private Product productFromSet(ResultSet resultSet) {
```

Comment:
```suggestion
    private Product getProductFromSet(ResultSet resultSet) {
```
don't forget about methods naming